### PR TITLE
Record all issues linked to a tab in summary.

### PR DESCRIPTION
**Pull request by michelle192837**
This helps us in two ways:

1) We know which issues are actually linked in a table, as opposed to issues that MIGHT be linked in issue state (with the bonus that it gives us the issues after filtering! :D )
2) We can display a list of all linked issues in the tab summaries on the Summary tab.

### CODE REVIEW

File: pkg/summarizer/summary.go [hunk 1]
```diff
@@ -522,6 +523,20 @@ func overallStatus(grid *statepb.Grid, recent int, stale string, brokenState boo
 	return summarypb.DashboardTabSummary_UNKNOWN
 }
 
+func allLinkedIssues(rows []*statepb.Row) []string {
+	issueMap := make(map[string]bool)
```
Comment (MushuEE) [inline pkg/summarizer/summary.go:527]: Is `IssueSet` a better name? or `uniqueIssues` 

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_identifier_length

in his guidelines he recommended not including type in the name.

File: pkg/summarizer/summary_test.go [hunk 2]
```diff
@@ -452,6 +455,70 @@ func TestRecentColumns(t *testing.T) {
...
+				},
+				{
+					Name:    "test-3",
+					Results: []int32{int32(statepb.Row_PASS), 10},
+				},
+			},
+			want: []string{},
+		},
+		{
+			name: "multiple linked issues",
```
Comment (MushuEE) [inline pkg/summarizer/summary_test.go:488]: Maybe also one for dupe issues?
```
		{
			name: "duplicate linked issues",
			rows: []*statepb.Row{
				{
					Name:    "test-1",
					Results: []int32{int32(statepb.Row_PASS), 10},
				},
				{
					Name:    "test-2",
					Results: []int32{int32(statepb.Row_PASS), 10},
                                   BugId:   []string{"1", "2"},
				},
				{
					Name:    "test-3",
					Results: []int32{int32(statepb.Row_PASS), 10},
                                   BugId:   []string{"2", "3"},
				},
			},
			want: []string{"1","2","3"},
		},
```

File: pkg/summarizer/summary.go (hunk 1 above)
Comment (michelle192837) [reply pkg/summarizer/summary.go:527]: I think issueSet helps describe what it's actually doing so using that!

**Comment by michelle192837** [comment]
Good suggestions, thanks! Done ^^

**Pull request merged**